src/test/scala/com/cognite/sdk/scala/v1/DataModelsTest.scala

```diff
@@ -124,31 +88,34 @@ class DataModelsTest extends CommonDataModelTestHelper with RetryWhile {
     Some(Seq(dataModel1.externalId))
   )
 
-  private val dataModels = Seq(dataModel1, dataModel2)
-
-  val expectedDataModelsOutputWithProps = dataModels.map { dm =>
-    dm.copy(properties =
-      dm.properties.map(x => x ++ Map("externalId" -> DataModelProperty("text", false)))
-    )
-  }
-
+  // toSeq is redundant on Scala 2.13, not Scala 2.12.
+  @SuppressWarnings(Array("org.wartremover.warts.RedundantConversions"))
   private def insertDataModels() = {
     val outputCreates =
       blueFieldClient.dataModels
         .createItems(Items[DataModel](Seq(dataModel1, dataModel2)))
         .unsafeRunSync()
         .toList
     outputCreates.size should be >= 2
+
+    retryWithExpectedResult[Seq[DataModel]](
+      blueFieldClient.dataModels.list().unsafeRunSync().toList.toSeq,
```

Comment:
Although reading this I'm wondering if we could just do `.toSeq` directly, no need for `.toList.toSeq` I guess? But maybe we still need the `SuppressWarnings`.